config/StoredProcedures/GetAllInventory.sql

```diff
@@ -30,4 +31,4 @@ on s1.ProductID = s2.ProductID;
 
 END $$
 
-DELIMITER ;
\ No newline at end of file
+DELIMITER ;
```

Comment:
The new sp returns the same result as the old one for our current testing data. But what if a run which has a larger ID has a smaller DateCreated. Using "MAX(R.DateCreated) AS LastRunDate, MAX(R.RunID) AS LastRun" together in the select statement, there is a possibility that the LastRunDate and the LastRunID do not come from the same run. I would recommend not change it.